tests/rkt_net_test.go

```diff
@@ -608,6 +608,10 @@ func testNetCustomNatConnectivity(t *testing.T, nt networkTemplateT) {
 
 	// Child connects to host
 	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatalf("os.Hostname() got error %q", err)
```

Comment:
@jonboulle Absolutely, and I've struggled with that particular problem before in other Go projects, but using panic usually makes debugging harder and we won't be using the features of t.Testing. 

My main concern is reducing the time between seeing the error and fixing it, I can do:
`panic(fmt.Errorf("os.Hostname() got error %q", err))`

Perhaps I should open a PR with a stack prettifier integration. It would be easy to replace in the future as well.